### PR TITLE
awslogs: add page

### PR DESCRIPTION
**Pull request by shano**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/awslogs.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`awslogs streams {{/var/log/syslog}}`
+
+- Get logs for any streams in the specified group between 1 and 2 hours ago:
+
+`awslogs get {{/var/log/syslog}} --start='{{2h ago}}' --end='{{1h ago}}'`
+
+- Get logs that match a specific Cloudwatch Logs Filter pattern:
+
+`awslogs get {{/aws/lambda/my_lambda_group}} --filter-pattern='ERROR'`
```
Comment (agnivade) [inline pages/common/awslogs.md:20]: I think ERROR should be in tokens(`{{}}`) too ?

Comment (shano) [reply pages/common/awslogs.md]: Ah yeah you're right @agnivade just added it!

**Comment by sbrl** [comment]
Also, next time, perhaps you could avoid force-pushing please? It makes it harder to tell what's changed when reviewing, and we've got a special button that allows us to squash easily on merge :smiley_cat:

**Comment by shano** [comment]
Ah okay no worries, will do @sbrl ! 😄

**Pull request merged**